src/proto/com/magnite/openrtb/v2/openrtb-xapi.proto

```diff
@@ -138,6 +138,13 @@ message ImpExt {
 
         // Floor data used to synthesize RTB rules.
         repeated Floor floors = 9;
+
+        // OD-17689 GPID needs to be handled by ImpExt_rp
+        // Global Placement ID. The unique identifier of the placement that is sent 
+        // to all participants in buying impressions for this placement. In general, 
+        // this field will be populated with one of the DFP Ad Unit Code, Prebid Ad Slot,
+        // or value provided by the publisher or ad server.
```

Comment:
This phrasing doesn't really make sense to me. It seems like it's talking about us populating it? This is for the incoming bid request, not something we have any control over—the value would always be populated by the publisher/ad server.

Reply:
Ah, didn't realize that. I guess that's ok then. I'd still suggest maybe changing in both comments "will" to "should" and "value" to "some value." The original PR (https://github.com/MagniteEngineering/xapi-proto/pull/4) has more context on the intention.

Reply:
done